### ISSUE TITLE
error during ./run.sh

### ISSUE THREAD

**Issue by bhomass** [ajitrajasekharan/bert_vector_clustering#1]
I executed the run.sh step, and got an error
inquired 0.51 0.0 ['inquired', 'asks']
Processing  28118  of  28996
***Singleton arr for term: MacKenzie

Has anyone seen this before? 
the debug_pivots.txt file size currently sits at 554231.

What is the correct size of debug_pivots.txt, when run to completion?

**Comment by ajitrajasekharan** [comment]
For bert-large-cased model I got an output of 
-rw-rw-r-- 1 deepcompute deepcompute 554073 Mar  8 02:43 debug_pivots.txt
and for wc -l debug_pivots.txt
6112 debug_pivots.txt
What is the model being used to cluster on?

**Comment by bhomass** [comment]
yes, it was bert-large-cased . looks like my debug_pivots.txt is one line longer than yours.

6113 debug_pivots.txt

**Comment by ajitrajasekharan** [comment]
Mine had the singleton array clusters removed. That could explain the difference. So the generation appears to have completed. Not sure what the error is. I will try , duplicate and get back
